spec/07-implicits.md

```diff
@@ -239,7 +239,7 @@ of types for which implicit arguments are searched is
 
 ```scala
 Throwable => Ordered[Throwable],
-Throwable => Ordered[Throwable],
+Throwable => Ordered[Throwable]
 ...
```

Comment:
if we interpret `...` as more elements to follow, we should keep the `,`.
Otherwise you could delete that also